tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py27,py36
+
+[testenv]
+deps = -rdev-requirements.txt
```

Comment:
would it work with `-r dev-requirements.txt` as well? Or is this tox specific?

Reply:
the parser doesn't care about spaces, works with both